app/src/AppRoutes.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { Suspense } from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import Loading from 'components/common/Loading';
+import { Layout } from 'components/layout';
+import { PUBLIC_URL } from 'config';
+
+const LazyAuthPage = React.lazy(() => import('components/auth/AuthPage'));
+const LazyLoopPage = React.lazy(() => import('components/loop/LoopPage'));
+const LazyHistoryPage = React.lazy(() => import('components/history/HistoryPage'));
+const LazyPoolPage = React.lazy(() => import('components/pool/PoolPage'));
+const LazySettingsPage = React.lazy(() => import('components/settings/SettingsPage'));
+const LazyConnectPage = React.lazy(() => import('components/connect/ConnectPage'));
+
+const AppRoutes: React.FC = () => {
+  return (
+    <Suspense fallback={<Loading delay={500} />}>
+      <BrowserRouter basename={PUBLIC_URL}>
+        <Routes>
+          <Route path="/" element={<LazyAuthPage />} />
+          <Route>
+            <Route
+              path="loop"
+              element={
+                <Layout>
+                  <LazyLoopPage />
+                </Layout>
+              }
+            />
+            <Route
+              path="history"
+              element={
+                <Layout>
+                  <LazyHistoryPage />
+                </Layout>
+              }
+            />
+            <Route
+              path="pool"
+              element={
+                <Layout>
+                  <LazyPoolPage />
+                </Layout>
+              }
+            />
+            {/* Only the GeneralSettings component needs hamburger menu 
+            hence not wrapping entire settings page in <Layout>. */}
+            <Route path="settings/*" element={<LazySettingsPage />} />
```

Comment:
We currently keep the sidebar (Layout comp) on all of the settings pages. Why do you think we should remove it? The styling breaks (black font) without the Layout parent component on the nested routes.

Reply:
Motive was to avoid overlapping icon.

<img width="855" alt="image" src="https://user-images.githubusercontent.com/6608672/170811996-570e1cd7-b047-4138-98a1-959154885400.png">

Was scratching my head as to what the styling issues you were referring to. Turns out _Dark Reader_ extension was enabled for `localhost:3000` so wasn't seeing the black font color.
I'll wrap settings in `Layout` and add some padding to the `←` icon.